mozilla/bedrock
# Fix #10736: Move careers content from lumbergh into bedrock

## Description

This moves all Django files, templates, and assets from mozilla/lumbergh into bedrock.

## Issue / Bugzilla link

Issue #10736 

## Testing

- Ensure all links to careers pages are updated to `/careers/`
- Ensure all content under `/careers` displays and works correctly
- Ensure the Wordpress blogs are imported and displayed correctly
- Ensure the Greenhouse job listings are imported and displayed correctly

Link to demo:
https://www-demo2.herokuapp.com/en-US/careers/
